### ISSUE TITLE
Latest version breaks Nuxt Devtools

### ISSUE THREAD

**Issue by peoray** [nuxt-modules/supabase#385]
### Version
@nuxtjs/supabase: v1.3.2 
nuxt: v3.12.3 

### Steps to reproduce
- Initialize a new nuxt project
- Add @nuxt/supabase

### What is Expected?

Nuxt Devtools should still be visible

### What is actually happening?

The devtools is not rendered and not visible

**Comment by jbjorge** [comment]
Maybe relevant context - I get this in the browser console:
```
SyntaxError: The requested module '/_nuxt/node_modules/.pnpm/@supabase+postgrest-js@1.15.8/node_modules/@supabase/postgrest-js/dist/cjs/index.js?v=469b7ed5' does not provide an export named 'default' (at wrapper.mjs:1:8)
```

**Comment by peoray** [comment]
@jbjorge thanks for the addition, I also got that same error

**Comment by larbish** [comment]
Are you sure you've updated to the `1.3.2`? The `'/_nuxt/node_modules/.pnpm/@supabase+postgrest-js@1.15.8/node_modules/@supabase/postgrest-js/dist/cjs/index.js?v=469b7ed5' does not provide an export named 'default' (at wrapper.mjs:1:8)`error was part of the `1.3.1`. My devtools is working as usual on my project with the latest version.

**Comment by jbjorge** [comment]
Looks to be 1.3.2 - and thanks for the quick follow up!
```
▶ pnpm why @nuxtjs/supabase
Legend: production dependency, optional only, dev only

devDependencies:
@nuxtjs/supabase 1.3.2
```

**Comment by thanhtung167** [comment]
how to fix have same error sir

**Comment by jbjorge** [comment]
Could the change to [optimizeDeps be to blame?](https://github.com/nuxt-modules/supabase/compare/v1.3.1...v1.3.2#diff-030fc083b2cbf5cf008cfc0c49bb4f1b8d97ac07f93a291d068d81b4d1416f70) 

It both lists `cookie` and `@supabase/postgrest-js`, and none of them are found by vite.
```
 WARN  Failed to resolve dependency: cookie, present in 'optimizeDeps.include'
 WARN  Failed to resolve dependency: @supabase/postgrest-js, present in 'optimizeDeps.include'
 ```

**Comment by larbish** [comment]
Could you please provide a reproduction? I'll have a look. I'm not able to reproduce myself.

**Comment by jbjorge** [comment]
[Reproduction repo here](https://github.com/jbjorge/nuxt-supa-reprod)
(I did not push .env file with SUPABASE_URL and SUPABASE_KEY, so those will need to be set)

Steps done to create the reproduction:
1. `pnpm dlx nuxi@latest init <project-name>`
2. pinned pnpm version to 9.5.0 in package.json (that thing was new to me, hope it doesn't cause trouble)
3. `npm run dev`
4. Verified that devtools work
5. Installed supabase module through the devtools UI

Post module install
* Terminal outputs
  *  WARN  Failed to resolve dependency: cookie, present in 'optimizeDeps.include'
  *  WARN  Failed to resolve dependency: @supabase/postgrest-js, present in 'optimizeDeps.include'

* Browser console outputs
  * Uncaught SyntaxError: The requested module '/_nuxt/node_modules/.pnpm/@supabase+postgrest-js@1.15.8/node_modules/@supabase/postgrest-js/dist/cjs/index.js?v=c0b4e737' does not provide an export named 'default' (at wrapper.mjs:1:8)

* Devtools in browser is not available/displayed/loaded

**Comment by peoray** [comment]
> Are you sure you've updated to the `1.3.2`? The `'/_nuxt/node_modules/.pnpm/@supabase+postgrest-js@1.15.8/node_modules/@supabase/postgrest-js/dist/cjs/index.js?v=469b7ed5' does not provide an export named 'default' (at wrapper.mjs:1:8)`error was part of the `1.3.1`. My devtools is working as usual on my project with the latest version. 

could also be related to the Nuxt version. What version are you using?

I'm using the latest Nuxt version

**Comment by jbjorge** [comment]
I think I've pinned it down to being a `pnpm` issue and not an issue with the supabase-module.
My reproduction repo works fine with npm.
Don't know if this is something that should be fixed by the supabase module or if we should add some config in pnpm.

**Comment by peoray** [comment]
> I think I've pinned it down to being a `pnpm` issue and not an issue with the supabase-module.
> My reproduction repo works fine with npm.
> Don't know if this is something that should be fixed by the supabase module or if we should add some config in pnpm.

does installing as a dependency or devdependency make any difference?

**Comment by peoray** [comment]
@jbjorge which pnpm version are you using

**Comment by larbish** [comment]
Should be fixed in `1.3.4` ✅

**Comment by larbish** [comment]
> does installing as a dependency or devdependency make any difference when using pnpm?

It does not change anything with Nuxt.

**Comment by peoray** [comment]
> Should be fixed in `1.3.4` ✅ 

awesome, thanks.

for some weird reason, it also breaks shadcn-vue as well (pnpm)

**Comment by CyberSekure** [comment]
Thanks for this, was scratching my head yesterday and just noticed this issue (I had the same). Fixed it for me!
